libs/arcade-cli/arcade_cli/usage/command_tracker.py

```diff
@@ -0,0 +1,305 @@
+import functools
```

Comment:
This is what wraps all CLI commands to track their execution without needing to explicitly track inside of command implementations